usecases/datalink_samples/README.md

```diff
@@ -0,0 +1,11 @@
+Two VOTABLE examples for service descriptor IRL templating
+
+GAIADR2CepheidParams.vot contains Cepheid variability parameters. A service is decribed which allows to retrieve the record of the associated DR2 source in the main catalog in various formats
+
+GAIADR2meanvelocities.vot contains mean radial velocities for for GAIA sources. 4 services are defined : one allows to retrieve each recird in another format,
+
+one allows to retrieve the source record in the main GAIA catalog

```

Comment:
These might be easier to read as dot points (use a leading - ):
`
- one allows to retrieve each record in another format,
- one allows to retrieve the source record in the main GAIA catalog`